From 482f85bf9ea59b949a072e8662bdcd70831841d4 Mon Sep 17 00:00:00 2001
From: Sean Hoyt <seanhoyt963@gmail.com>
Date: Fri, 30 Dec 2016 16:08:49 -0600
Subject: [PATCH] Remove RIL_onRequestAck

Revert this if we get a newer device that actually uses this

Change-Id: Id3b0e1fe3a0e778e2c5252418500a96d4b019172
---
 rild/rild.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/rild/rild.c b/rild/rild.c
index ce1adba..004a5f7 100644
--- a/rild/rild.c
+++ b/rild/rild.c
@@ -58,8 +58,6 @@ extern void RIL_register_socket (RIL_RadioFunctions *(*rilUimInit)
 extern void RIL_onRequestComplete(RIL_Token t, RIL_Errno e,
         void *response, size_t responselen);
 
-extern void RIL_onRequestAck(RIL_Token t);
-
 extern void RIL_setRilSocketName(char *);
 
 #if defined(ANDROID_MULTI_SIM)
@@ -77,8 +75,7 @@ extern void RIL_requestTimedCallback (RIL_TimedCallback callback,
 static struct RIL_Env s_rilEnv = {
     RIL_onRequestComplete,
     RIL_onUnsolicitedResponse,
-    RIL_requestTimedCallback,
-    RIL_onRequestAck
+    RIL_requestTimedCallback
 };
 
 extern void RIL_startEventLoop();